paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -121,6 +124,15 @@ public class PaparazziPlugin @Inject constructor(
       val reportOutputDir = project.extensions.getByType(ReportingExtension::class.java).baseDirectory.dir("paparazzi/${variant.name}")
       val failedSnapshotOutputDir = buildDirectory.dir("paparazzi/failures")
 
+      val testInstrumentation = testVariant.instrumentation
+      testInstrumentation.transformClassesWith(
+        ResourcesCompatVisitorFactory::class.java,
+        InstrumentationScope.ALL
+      ) { }
+      testInstrumentation.setAsmFramesComputationMode(
+        FramesComputationMode.COMPUTE_FRAMES_FOR_INSTRUMENTED_METHODS
+      )
+
```

Comment:
nit: this is OK in the short term to support AndroidX fonts, but this adds more layers to difficulty to:
* any hopes of a Bazel plugin
* SDKifying

I'm less concerned about the first bullet point (although I try my best to be mindful of it in design decisions like this).  The second one makes me more sad, cause we know we have to tackle this soon-ish anyway.

In a follow-up, we should follow in Android Studio's footsteps and move this asm stuff into some classloader shim.

Reply:
> I'm less concerned about the first bullet point (although I try my best to be mindful of it in design decisions like this). The second one is more top of mind for me, cause we know we'll have to tackle this soon-ish anyway.
> 
> In a follow-up, we should follow in Android Studio's footsteps and move this asm stuff into some classloader shim.

Yep 100% that relying on gradle more isn't the final way to do this. I think we can look into the ASM class loading that way we can remove the reliance on ByteBuddy shims and implement these asm transforms at runtime.

Reply:
Yep, that's precisely been the goal!